docs/send-data/kubernetes/collecting-logs.md

```diff
@@ -504,6 +504,141 @@ sumologic:
+
+:::note
+Remember to use [attributes after translation to Sumo Logic schema](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/sumologicprocessor#attribute-translation). For example, use `namespace` instead of `k8s.namespace.name`.
+:::
+
+:::note
+If a log matches multiple statements, it will be sent to all corresponding exporters. For example, the following routing table will send logs from the `production` namespace to `sumologic/production` and `sumologic/soc`:
+
+```yaml
+- exporter: sumologic/soc
+  statement: route() where resource.attributes["namespace"] == "production"
+- exporter: sumologic/production
+  statement: route() where resource.attributes["namespace"] == "production"
+```
+
+:::
+
+### Send logs selectively to multiple locations
+
+This scenario differs from the previous ones by not sending all logs to Sumo Logic. If you want to send some data to Sumo Logic and some (the same or other) data to other locations, you need to disable default exporters. Set `sumologic.logs.otelcol.useDefaultExporters` to `false,` and then set `sumologic.logs.otelcol.routing.fallbackExporters` to a list of exporters that are going to be used for the logs which do not match any of the routing table statements.
+
+In the following example, logs from the `production` namespace are going to be sent to the production Sumo Logic exporter, and all remaining logs will be sent to the default Sumo Logic exporter:
+
+```yaml
+sumologic:
+  logs:
+    otelcol:
+      ## useDefaultExporters set to false to stop sending all logs to default exporters
+      useDefaultExporters: false
+      extraExporters:
+        ## define `sumologic/production` exporter
+        sumologic/production:
+          ## use environmental variable to set endpoint
+          endpoint: ${PRODUCTION_ENDPOINT}
+      routing:
+        ## use sumologic as fallbackExporter, which means it will get all data which do not match any statement
+        fallbackExporters:
+          - sumologic
+        table:
+          ## send all logs from `production` namespace to `sumologic/production` exporter
+          - exporter: sumologic/production
+            statement: route() where resource.attributes["namespace"] == "production"
+metadata:
+  logs:
+    statefulset:
+      extraEnvVars:
+        ## Set `PRODUCTION_ENDPOINT` env variable for `sumologic/production exporter` as `endpoint-production-logs` property of `sumologic-secrets` secret
+        - name: PRODUCTION_ENDPOINT
+          valueFrom:
+            secretKeyRef:
+              name: sumologic-secrets
+              key: endpoint-production-logs
+```
+
+For the above example `sumologic-secrets` secret with `endpoint-production-logs` key in the Sumo Logic installation namespace is required.
```

Comment:
```suggestion
For the example above, the `sumologic-secrets` secret with the `endpoint-production-logs` key is required in the Sumo Logic installation namespace.
```

Reply:
I applied it for two the same occurrences as well

Reply:
thank you!